lib/much-rails/date.rb

```diff
@@ -0,0 +1,49 @@
+require "much-rails"
+
+# @example
+#   MuchRails::Date.for(nil) # => nil
+#   MuchRails::Date.for(" ") # => nil
+#   MuchRails::Date.for(Time.zone.today) # => Date
+#   MuchRails::Date.for(Time.current) # => Date
+#   MuchRails::Date.for(DateTime.current) # => Date
+#   MuchRails::Date.for("07/04/2020") # => Date
+#   MuchRails::Date.for("2020.07.04") # => Date
+#   MuchRails::Date.for("2020-07-04T08:15:00Z") # => Date
+module MuchRails; end
```

Comment:
Hmm...maybe these examples should go above the `self.for` method??

Reply:
Yeah, that might be good actually. 👍🏻